pkgs/by-name/pl/plex-desktop/package.nix

```
@@ -122,7 +122,7 @@ let
 in
 buildFHSEnv {
   inherit meta;
-  name = "${pname}-${version}";
+  name = "${pname}";
```

```suggestion
  name = pname;
```

``inherit meta`` should be last in the derivation.